app/application/use_cases/send_unlock_command.py

```
@@ -0,0 +1,143 @@
+        # Sign the command (HMAC-SHA256)
+        signature = self._sign_command(command_data)
+        command_data["signature"] = signature
+
+        # 5. Send command via WebSocket
+        message = {
+            "type": MessageType.COMMAND.value,
+            "payload": command_data,
+        }
+
+        try:
+            asyncio.create_task(manager.send_to_vault(vault_id, message))
```

Ruff (0.14.14)

[warning] Store a reference to the return value of `asyncio.create_task`

(RUF006)